plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -152,5 +153,29 @@ public static SymbolLookup get() {
         return j.getInjector().getInstance(SymbolLookup.class);
     }
 
+    /**
+     * Get the {@link Symbol} value for the class of the given object, if the annotation is present
+     *
+     * @param o An object
+     * @return The {@link Symbol} annotation value for the class that object represents, or null if the annotation is not present.
+     */
+    public static String getSymbolValue(Object o) {
+        return getSymbolValue(o.getClass());
+    }
+
+    /**
+     * Get the {@link Symbol} value for the given class, if the annotation is present
+     *
+     * @param c A class.
+     * @return The {@link Symbol} annotation value for the given class, or null if the annotation is not present.
+     */
+    public static String getSymbolValue(@Nonnull Class<?> c) {
+        Symbol s = c.getAnnotation(Symbol.class);
+        if (s != null && s.value()[0] != null) {
```

Comment:
:bug: null is never permitted in annotation values, so it is misleading to check for it.

What you _did_ want to check for is that `s.value().length > 0`, at least until we have an `AnnotationProcessor` that prevents you from compiling `@Symbol({})`.